crates/libcontainer/src/process/fork.rs

```diff
@@ -166,13 +166,11 @@ fn clone(cb: CloneCb, flags: u64, exit_signal: Option<u64>) -> Result<Pid, Clone
     let child_stack = unsafe {
         // Since nix = "0.27.1", `mmap()` requires a generic type `F: AsFd`.
         // `::<File>` doesn't have any meaning because we won't use it.
-        mman::mmap::<File>(
+        mman::mmap_anonymous(
             None,
             NonZeroUsize::new(default_stack_size).ok_or(CloneError::ZeroStackSize)?,
             mman::ProtFlags::PROT_READ | mman::ProtFlags::PROT_WRITE,
             mman::MapFlags::MAP_PRIVATE | mman::MapFlags::MAP_ANONYMOUS | mman::MapFlags::MAP_STACK,
```

Comment:
It seems we don't need `mman::MapFlags::MAP_ANONYMOUS` anymore.

Reply:
yes. i have also confirmed that `mmap_anonymous` has this line

```rust
pub unsafe fn mmap_anonymous(
    addr: Option<NonZeroUsize>,
    length: NonZeroUsize,
    prot: ProtFlags,
    flags: MapFlags,
) -> Result<NonNull<c_void>> {
    // ...

    let flags = MapFlags::MAP_ANONYMOUS | flags;

    // ...
}
```

Reply:
Yep that is redundant now. Will remove it. Thanks !